web/controllers/certification_controller.ex

```diff
@@ -63,37 +63,41 @@ defmodule Vae.CertificationController do
       |> Map.put_new(:administrative, Plug.Conn.get_session(conn, :search_administrative))
```

Comment:
Can you remove `Plug.Conn.get_session(conn, ....)` from controller and use `get_session(conn, ...)` directly